app/javascript/packs/contact-form-navigation-alert.js

```diff
@@ -0,0 +1,23 @@
+$(() => {
+   $("body").attr("data-turbolinks", false);
```

Comment:
What's this for?

Reply:
Turbo links in rails change pages without causing a full page reload. As it turns out, the onbeforeunload works only with full page reloads. When setting data-turbolinks attribute to false, we disable the turbolink behavior only for the specific page. When we leave it this change will disappear.